src/main/cpp/mechanisms/controllers/ControlDataXmlParser.cpp

```diff
@@ -148,7 +159,7 @@ ControlData *ControlDataXmlParser::ParseXML(
     }
     if (!hasError)
     {
-        data = new ControlData(mode, server, identifier, p, i, d, f, izone, maxAccel, cruiseVel, peak, nominal);
+        data = new ControlData(mode, server, identifier, p, i, d, f, fType, izone, maxAccel, cruiseVel, peak, nominal, false);
```

Comment:
Should this be parsing the enableFOC option as well?  Or is that in the motor xml?

Reply:
This file is going away (hopefully) as the code generator is supposed to do this.  I only made these changes to make it compile.